packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -1067,13 +1096,52 @@ public void resumePreview() {
         null, (code, message) -> dartMessenger.sendCameraErrorEvent(message));
   }
 
-  public void startPreview() throws CameraAccessException {
+  public void startPreview() throws CameraAccessException, InterruptedException {
+    // if we are already recording, we are flipping the camera and must
```

Comment:
Nit: Please don't use "we" in code comments, since it's unclear who "we" is, making the comments harder to understand—here for instance the two uses of "we" appear to be different subjects. Use a clear subject, or if there isn't one us passive voice (e.g., "If recording is already in progress, the camera is being flipped, so send it through the VideoRenderer to keep the correct orientation.")

This applies throughout the PR.